website/source/docs/platform/k8s/run.html.md

```diff
@@ -0,0 +1,234 @@
+---
+layout: "docs"
+page_title: "Running Consul - Kubernetes"
+sidebar_current: "docs-platform-k8s-run"
+description: |-
+  Consul can run directly on Kubernetes, both in server or client mode. For pure-Kubernetes workloads, this enables Consul to also exist purely within Kubernetes. For heterogeneous workloads, Consul agents can join a server running inside or outside of Kubernetes.
+---
+
+# Running Consul on Kubernetes
+
+Consul can run directly on Kubernetes, both in server or client mode.
+For pure-Kubernetes workloads, this enables Consul to also exist purely
+within Kubernetes. For heterogeneous workloads, Consul agents can join
+a server running inside or outside of Kubernetes.
+
+This page starts with a large how-to section for various specific tasks.
+To learn more about the general architecture of Consul on Kubernetes, scroll
+down to the [architecture](/docs/platform/k8s/run.html#architecture) section.
+
+## Helm Chart
+
+The recommended way to run Consul on Kubernetes is via the
+[Helm Chart](/docs/platform/k8s/helm.html). This will install and configure
+all the necessary components to run Consul. The configuration enables you
+to run just a server cluster, just a client cluster, or both. Using the Helm
+Chart, you can have a full Consul deployment up and running in seconds.
+
+While the Helm Chart exposes dozens of useful configurations and automatically
+sets up complex resources, it **does not automatically operate Consul.**
+You are still reponsible for learning how to monitor, backup,
+upgrade, etc. the Consul cluster.
+
+The Helm Chart has no required configuration and will install a Consul
+cluster with sane defaults out of the box. Prior to going to production,
+it is highly recommended that you
+[learn about the configuration options](/docs/platform/k8s/helm.html#configuration-values-).
+
+## How-To
+
+### Installing Consul
+
+To install Consul, clone the consul-helm repository, checkout the latest release, and install
+Consul. You can run `helm install` with the `--dry-run` flag to see the
+resources it would configure. In a production environment, you should always
+use the `--dry-run` flag prior to making any changes to the Consul cluster
+via Helm.
+
+```sh
+# Clone the chart repo
+$ git clone https://github.com/hashicorp/consul-helm.git
+$ cd consul-helm
+
+# Checkout a tagged version
+$ git checkout v0.1.0
+
+# Run Helm
+$ helm install --name consul ./
+...
+```
+
+_That's it._ The Helm chart does everything to setup a recommended
+Consul-on-Kubernetes deployment.
```

Comment:
Just a thought: I'm guessing we don't default the raft performance numbers in helm right? In general those are optimized for OOBE and not great values in production if you have decent sized servers.

We've made it super clear that the helm defaults are also optimised for OOBE, but I wonder if we need to be careful about setting expectations here that they should go to prod with this "recommended" setup? We usually suggest that anyone running Consul in prod should read and tune the raft performance knobs at least.